src/sentry/api/endpoints/organization_pinned_searches.py

```diff
@@ -14,6 +14,9 @@
 class OrganizationSearchSerializer(serializers.Serializer):
     type = serializers.IntegerField(required=True)
     query = serializers.CharField(required=True)
+    sort = serializers.ChoiceField(
+        choices=SortOptions.as_choices(), default=SortOptions.DATE, required=False
```

Comment:
Is the plan to make this required once the frontend is passing this all the time?

Reply:
we did this for the saved search api to make it not a breaking api change, repeated that here